.github/workflows/arch_linux_test.yml

```diff
@@ -0,0 +1,26 @@
+name: Arch Linux Installation Test
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container: archlinux:latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: |
+        pacman -Syu --noconfirm
+        pacman -S --noconfirm git zsh curl sudo
+    - name: Run installation script
+      run: |
+        chmod +x ./installOhMyZSH.sh
```

Comment:
I might be wrong, but you might need to clone the repo first

Reply:
Ooo right. I will change that.